src/main/java/com/mule/einstein/internal/models/ParamsEmbeddingDocumentDetails.java

```diff
@@ -1,23 +1,15 @@
-package com.mule.einstein.internal.helpers.documents;
+package com.mule.einstein.internal.models;
 
-import com.mule.einstein.internal.models.provider.EmbeddingModelApiNameProvider;
+import com.mule.einstein.internal.helpers.documents.DocumentFileType;
+import com.mule.einstein.internal.helpers.documents.DocumentSplitOptions;
 import org.mule.runtime.api.meta.ExpressionSupport;
 import org.mule.runtime.extension.api.annotation.Expression;
 import org.mule.runtime.extension.api.annotation.param.Optional;
 import org.mule.runtime.extension.api.annotation.param.Parameter;
 import org.mule.runtime.extension.api.annotation.param.display.DisplayName;
 import org.mule.runtime.extension.api.annotation.values.OfValues;
 
-import static com.mule.einstein.internal.helpers.ConstantUtil.OPENAI_ADA_002;
-
-public class ParametersEmbeddingDocument {
-
-  @Parameter
-  @Expression(ExpressionSupport.SUPPORTED)
-  @OfValues(EmbeddingModelApiNameProvider.class)
-  @Optional(defaultValue = OPENAI_ADA_002)
-  @DisplayName("Model API Name")
-  private String modelApiName;
+public class ParamsEmbeddingDocumentDetails extends ParamsEmbeddingModelDetails {
```

Comment:
checking on the UI front and then can remove . This was done to avoid duplicacy and keep the namings and display consistent on all places

Reply:
![image](https://github.com/user-attachments/assets/a63f46b1-8221-4c0b-950c-b1e8d6fed87e)

Looks fine in UI

Reply:
There are few other classes where we are using  modelApiName. Creating this class and using it here only is not making any sense. Ex- RAGParamsModelDetails has modelApiName field, we cannot extend this class in RAGParamsModelDetails, until we are agree to change the fields position

Reply:
Discussed in-person and agreed, this will remain as-is. RAGParamsModelDetails is different and has different value provider and default value. Hence not applicable in this scenario.
keeping this change for now